Userland/Applications/Browser/EditBookmarkDialog.h

```diff
@@ -7,34 +7,34 @@
 #pragma once
 
 #include "EditBookmarkWidget.h"
+#include <LibGUI/Dialog.h>
 #include <LibGUI/TextBox.h>
 #include <LibGUI/Window.h>
-#include <LibGUI/Dialog.h>
 
 namespace Browser {
 
-    enum class PerformEditOn {
-        NewBookmark,
-        ExistingBookmark
-    };
+enum class PerformEditOn {
```

Comment:
Regarding the whole commit: Please make sure that `clang-format` runs properly after each individual commit, and drop the all-in-one fix that has been attached at the end.